addon/components/frost-object-browser.js

```diff
@@ -22,6 +22,9 @@ export default Ember.Component.extend({
   classNames: ['frost-object-browser'],
   contentHeight: 505,
   detailLevel: 'low',
+  isHighDetailVisible: true,
+  isLowDetailVisible: true,
+  isMediumDetailVisible: false,
```

Comment:
Curious. Why do we need extra attribute to achieve this? Couldn't you figure out whether the low, medium, and/or high LOD schemas were passed in? Or is the requirement to pass in them all and dynamically hide/show them based on something else?